lib/environment.rb

```diff
@@ -164,6 +164,17 @@ def admin_console_url
       return @admin_console_url
     end
 
+    def openshift_idp
+      unless @openshift_idp
+        if opts[:openshift_idp]
+          @openshift_idp = opts[:openshift_idp]
+        else
+          @openshift_idp = "kube:admin"
```

Comment:
If no `openshift_idp` is defined `kube:admin` will be used, usually the first one